### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by Julia-elsammak**
Added cleanup comment
Added Manifest Templates

### What does this PR do? What is the motivation?
Added a note regarding the priority of cleanup steps
Added Manifest Templates for an easy reference

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/containers/kubernetes/installation.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ The following command deletes all the Kubernetes resources created by the above
 kubectl delete datadogagent datadog
 helm delete my-datadog-operator
 ```
-
+Please ensure to remove DatadogAgent objects from your cluster first and then uninstall the operator via helm uninstall
```
Comment (maycmlee) [inline content/en/containers/kubernetes/installation.md:198]:
```suggestion
Make to remove DatadogAgent objects from your cluster first and then uninstall the operator using the helm uninstall command.
```

File: content/en/containers/kubernetes/installation.md [hunk 2]
```diff
@@ -295,6 +295,12 @@ If Google Container Registry ([gcr.io/datadoghq][8]) is not accessible in your d
 - **Configure integrations**. See [Integrations & Autodiscovery][5].
 - **Other configurations**: To collect events, override proxy settings, send custom metrics with DogStatsD, configure container allowlists and blocklists, or reference the full list of available environment variables, see [Further Kubernetes Configuration][4].
 
+Manifest Templates
```
Comment (maycmlee) [inline content/en/containers/kubernetes/installation.md:298]:
```suggestion
### Manifest Templates
```

File: content/en/containers/kubernetes/installation.md [hunk 3]
```diff
@@ -295,6 +295,12 @@ If Google Container Registry ([gcr.io/datadoghq][8]) is not accessible in your d
...
+
+[Manifest with Logs, APM, process, and metrics collection enabled]
+[Manifest with Cluster Agent].
```
Comment (maycmlee) [inline content/en/containers/kubernetes/installation.md:301]:
```suggestion
[Manifest with logs, APM, processes, and metrics collection enabled].
[Manifest with the Cluster Agent].
```

Comment (maycmlee) [reply content/en/containers/kubernetes/installation.md:301]: Where should these be linked to?

**Review by maycmlee — COMMENTED** [review]
Left some suggestions and a question about the manifests.

**Comment by Julia-elsammak** [comment]
Will create a new one

**Pull request closed without merging**